applications/grafana-metrics/chart/Chart.yaml

```diff
@@ -15,12 +15,12 @@ sources:
   - https://github.com/FIWARE-Ops/marinera/applications/grafana-metrics
 dependencies:
 - name: grafana
-  version: 6.28.0 # Grafana 8.5.0
+  version: 6.29.3
```

Comment:
Isnt this a bit risky to change the dependency at this stage?

I have seen it working in the NS though...